internal/widget/scroller.go

```diff
@@ -538,16 +442,13 @@ func (s *Scroll) Refresh() {
 	s.refreshWithoutOffsetUpdate()
 }
 
-func (s *Scroll) refreshWithoutOffsetUpdate() {
-	RefreshWidget(s)
+func (s *Scroll) Resize(sz fyne.Size) {
+	s.Base.Resize(sz)
+	s.Refresh()
```

Comment:
Shouldn’t we keep the refreshes limited to actual changes of the size (`size != s.size`)?

Reply:
Yes, good catch thanks.